### DISCUSSION TITLE
Support for Redshift SQL ETL?

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#14108] [General]
Hi,

Does RavenDB Cloud support SQL ETL to AWS Redshift. Redshift seems to have its own ODBC drivers and recommends against using PostgreSQL driver.

Is there a working solution for ETL to Redshift?

Thanks

**Comment by ayende** [comment]
I assume that you mean this? 
https://docs.aws.amazon.com/redshift/latest/dg/c_redshift-postgres-jdbc.html

That isn't relevant for RavenDB and I think that you should be able to connect properly as Postgres destination using SQL ETL with no issue.
Note that the features that SQL ETL needs are compatible with Redshift.

For big OLAP tasks, we also offer OLAP ETL - which is typically paired with AWS Athena, mind.

**Comment by nickchampion** [comment]
Thanks Oren, we'll try using PostgreSQL destination and see how we get on
